internal/proto/data_coord.proto

```diff
@@ -231,7 +231,7 @@ message SaveBinlogPathsRequest {
   repeated SegmentStartPosition start_positions = 6;
   bool flushed = 7;
   repeated FieldBinlog field2StatslogPaths = 8;
-  repeated DeltaLogInfo deltalogs = 9;
+  repeated Binlog deltalogs = 9;
```

Comment:
Just wondering, is it better to also FieldBinlog for delta logs and make Binlog a private concept inside FieldBinLog?

Reply:
or we simply use the concept binlog and make field as a optional parameter.

Reply:
> Just wondering, is it better to also FieldBinlog for delta logs and make Binlog a private concept inside FieldBinLog?

It's doable but `FieldID` will be redundant as each Binlog for same column will all have the same Field ID value.

The reason why DeltaLogs does not need field ID is that the delete entry is always applied on PK field.

It true that it looks ugly here. I will unify the `deltalogs` like other binlog fields.